content/blog/2020-10-26-october-20-community-gems.md

```diff
@@ -0,0 +1,163 @@
+storage credentials to everyone on your team, but you still want to check-in
+your DVC setup (meaning, your `.dvc/config` file). In this case, you want to use
+a `local` config file!
+
+You can use the command
+
+```dvc
+$ dvc config --local
+```
+
+to setup remote credentials that will be stored in `.dvc/config.local`- by
+default, this file is in your `.gitignore` so you don't have to worry about
+accidentally committing secrets to your Git repository.
+[Check out the docs](https://dvc.org/doc/command-reference/config) for more.
+
+## CML Questions
+
+### [Q: What's the file size limit for publishing files with `cml-publish`?](https://discordapp.com/channels/485586884165107732/728693131557732403/751001285100306502)
+
+`cml-publish` is a service for hosting files that are embedded in CML reports,
+like images, audio files, and GIFS. By default, we have a limit of 2 MB per
+upload.
+
+If your files are larger than this (which can happen, depending on the machine
+learning problem you're working on!) we recommend using GitLab's artifact
+storage.
+[Based on discussions in the community](https://github.com/iterative/cml/issues/232),
+we recently implemented a CML flag (`--gitlab-uploads`) to streamline the
+process:
+
+```dvc
+$ cml-publish movie.mov --md --gitlab-uploads > report.md
+```
+
+Note that we don't currently have an analagous solution for GitHub, because
+GitHub artifacts expire after 90 days (whereas they're permanent in GitLab).
+
+### [Q: I'm getting a mysterious error message, `Failed guessing mime type of file`, when I try to use `cml-publish`. What's going on?](https://discordapp.com/channels/485586884165107732/728693131557732403/763840404675756042)
+
+This error message usually means that the target of `cml-publish`- for example,
+
+```dvc
+$ cml-publish <target file>
+```
+
+is not found. Check for typos in the target filename and ensure that the file
+was in fact generated during the run (if it isn't part of your Git repository).
+We've [opened an issue](https://github.com/iterative/cml/issues/308) to add a
+more informative error message in the future.
+
+### [Q: In my GitHub Actions workflow, I use `dvc metrics diff` to compare metrics generated during the run to `main` and print a table- but the table isn't showing any of the metrics from `main`. What could be happening?](https://discordapp.com/channels/485586884165107732/728693131557732403/768815157034876929)
+
+When a continuous integration runner won't report metrics from previous versions
+of your project (or other branches), that's usually a sign that the runner
+doesn't have access to the full Git history of your project or your metrics
+themselves. Here are a few things to check for:
+
+1. **Did you fetch your Git working tree in the runner?** Functions like
+   `dvc metrics diff` require the Git history to be accessible- make sure that
+   in your workflow, before you run this function, you've done a `git fetch`. We
+   recommend
+
+```dvc
```

Comment:
you should indent this probably, to make it part of the list item, otherwise we have two 1s now

Reply:
yep